docker/package/packages.py

```diff
@@ -431,6 +431,8 @@ def mk_node_unit(
         target_networks=networks.keys(),
         network_protos=networks_protos,
         meta=packages_meta,
+        additional_native_deps=[f"tezos-baker-{proto}" for proto in active_protocols]
+        + ["tezos-node"],
```

Comment:
Why not add `acl` here too?

It seems that we do so for `tezos-baker-*` processes, so it would be nice to be consistent.

Reply:
but we already depend on `tezos-baker-*`, thus there is no need in duplication

Reply:
Oh, but what I mean is that we already do, except it's added to the list in (two places in) `model.py`.

Which BTW I think it's a good idea to have, to make it clear that it's a dependency for this package too.